src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -43,7 +43,8 @@ protected Object[][] getContents() {
             {"R_lengthTruncated", " The inserted length is truncated or not correct!"},
             {"R_timeValueTruncated", " The time value is truncated or not correct!"},
             {"R_invalidErrorMessage", "Invalid Error Message: "},
-            {"R_kerberosNativeGSSFailure", "No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)"},
+            {"R_kerberosNativeGSSFailure",
+                    "No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)"},
```

Comment:
Not part of this PR, but while you're here please change.
```suggestion
                    "No valid credentials provided (Mechanism level: Failed to find any Kerberos target)"},
```